data/rulesets.ts

```diff
@@ -310,7 +310,7 @@ export const Formats: {[k: string]: FormatData} = {
 			];
 			const species = this.dex.getSpecies(set.species || set.name);
 			if (!tundraDex.includes(species.baseSpecies) && !tundraDex.includes(species.name) &&
-				this.ruleTable.isBannedSpecies(species)) {
+				!this.ruleTable.has('+' + species.id)) {
 				return [`${species.baseSpecies} is not in the Crown Tundra Pokédex.`];
 			}
```

Comment:
this isnt actually how the ruletable works, i would just remove the third check entirely

Reply:
Ah, I just copied from all the other pokedex rulesets. Should I remove it from those too?

Reply:
The other rulesets aren't relevant to to this pull request, I would just open a separate PR to properly check validation after this one

Reply:
Cheers. I'm not sure I'm familiar enough with the ruletable to "properly check validation" in all the other rulesets, but I removed the third check from this for now!